src/Sylius/Bundle/AdminBundle/Resources/views/Product/Form/_form.html.twig

```diff
@@ -0,0 +1,7 @@
+{% set simple = constant('Sylius\\Component\\Product\\Model\\ProductInterface::SIMPLE') %}
+
+{% if product.type == simple %}
```

Comment:
Can we make `isSimple` and `isConfigurable` methods on a product, so it will just be `product.simple` and `product.configurable` checks?